src/runtime-rs/crates/runtimes/virt_container/src/factory/template/mod.rs

```diff
@@ -0,0 +1,172 @@
+// Copyright 2025 Kata Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::fmt::Debug;
+use std::fs::File;
+use std::path::PathBuf;
+
+use std::thread::sleep;
+use std::time::Duration;
+
+use anyhow::Context;
+use anyhow::{anyhow, Result};
+
+use async_trait::async_trait;
+use nix::mount::{mount, MsFlags};
+
+use crate::factory::vm::{VMConfig, VM};
+use kata_types::config::TomlConfig;
+
+const TEMPLATE_WAIT_FOR_AGENT: Duration = Duration::from_secs(2);
+const TEMPLATE_DEVICE_STATE_SIZE_MB: u32 = 8; // as in Go templateDeviceStateSize
```

Comment:
Please add a comment to explain what are they used for. Don't say it's copied from the Go runtime.